Shared/Samples/Group layers together/GroupLayersTogetherView.swift

```diff
@@ -0,0 +1,167 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct GroupLayersTogetherView: View {
+    /// A scene with an imagery basemap and a world elevations source.
+    @State private var scene: ArcGIS.Scene = {
+        let scene = Scene(basemapStyle: .arcGISImagery)
+        
+        // Add base surface to the scene for elevation data.
+        let elevationSource = ArcGISTiledElevationSource(url: .worldElevationService)
+        let surface = Surface()
+        surface.addElevationSource(elevationSource)
+        scene.baseSurface = surface
+        return scene
+    }()
+    
+    /// The current viewpoint of the scene view.
+    @State private var viewpoint: Viewpoint?
+    
+    /// A Boolean value that indicates whether the layers sheet is showing.
+    @State private var isShowingLayersSheet = false
+    
+    var body: some View {
+        SceneView(scene: scene, viewpoint: viewpoint)
+            .onViewpointChanged(kind: .centerAndScale) { viewpoint = $0 }
+            .task {
+                // Add group layers to the scene as operational layers.
+                scene.addOperationalLayers([makeProjectAreaGroupLayer(), makeBuildingsGroupLayer()])
+                
+                // Ensure all group layers' childlayers are loaded.
+                for groupLayer in scene.operationalLayers as! [GroupLayer] {
+                    await groupLayer.layers.load()
+                }
+                
+                // Set the scene's viewpoint with the extent of the project area group layer.
+                if let extent = scene.operationalLayers.first?.fullExtent {
+                    let camera = Camera(lookingAt: extent.center, distance: 700, heading: 0, pitch: 60, roll: 0)
+                    viewpoint = Viewpoint(latitude: .nan, longitude: .nan, scale: .nan, camera: camera)
+                }
+            }
+            .toolbar {
+                ToolbarItem(placement: .bottomBar) {
+                    Button("Layers") {
+                        isShowingLayersSheet = true
+                    }
+                    .sheet(isPresented: $isShowingLayersSheet, detents: [.medium]) {
+                        layersList
+                    }
```

Comment:
The settings sheet covers most of the layers on iPhone. The layer visibility can be fixed by refactoring the button and sheet using [fraction](https://developer.apple.com/documentation/swiftui/presentationdetent/fraction%28_%3A%29) presentation detents. Here is an [example](https://github.com/Esri/arcgis-maps-sdk-swift-samples/blob/main/Shared/Samples/Add%20dynamic%20entity%20layer/AddDynamicEntityLayerView.swift#L73-L97) using the fraction presentation detents.

<img src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/117859673/94f422cf-a439-4c2b-8c91-87de070a5ec3" width="300">

Reply:
Thanks for the feedback! I added the fraction presentation, but I am not sure if it is what you had in mind, so let know if it needs more work.